src/site/markdown/configuring.md

```diff
@@ -0,0 +1,100 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+# Configuring Apache Maven
+
+## System properties and JVM
+
+The `MAVEN_OPTS` environment variable contains parameters used to 
+start up the JVM running Maven and can be used to supply additional 
+options to it. For example, JVM memory settings could be defined 
+with the value `-Xms256m -Xmx512m`.
+
+JVM options can also be configured in the 
+`${session.rootDirectory}/.mvn/jvm.config` file, which means you can 
+define the options for your build on a per-project basis. This file 
+will become part of your project and will be checked in along with 
+your project.
+
+So for example if you put the following JVM options into the 
+`.mvn/jvm.config` file:
+```
+-Xmx2048m -Xms1024m -XX:MaxPermSize=512m -Djava.awt.headless=true
+```
+
+The content of this file will be appended to the `MAVEN_OPTS` 
+environment variable.
+
+Note that the above mechanism is handled by the Maven launch 
+scripts `mvn` and `mvnDebug` (`mvn.cmd` and `mvnDebug.cmd` on 
+Windows platforms).
+
+## User Properties
+
+Once the JVM is started, Maven uses _user properties_ to configure
+various parts of the system. The main entry point is the 
+`${maven.home}/conf/maven.properties` properties file. This file,
+along with any other files loaded from it (see below) are called
+_Maven properties files_ and are Java properties files with a 
+few enhancements. Contrary to standard properties file, those files 
+are UTF-8 encoded.
```

Comment:
This is a bit problematic, because Java properties files are UTF-8 now for quite some time. Please check. This may be misleading especially we are on Java 17 already.

Reply:
> This is a bit problematic, because Java properties files are UTF-8 now for quite some time. Please check. This may be misleading especially we are on Java 17 already.

https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Properties.html

XML properties files are encoded by default in UTF-8.  When using the text syntax, it's still ISO 8859-1 (look for `ISO 8859-1` in the above page).

Reply:
We never used XML style, right? This doesn't apply. I am talking about text style properties file. I think I was confused by ResourceBundles: https://openjdk.org/jeps/226, not properties files per sé.